synapse/handlers/event_auth.py

```diff
@@ -156,7 +156,7 @@ async def get_user_which_could_invite(
             Codes.UNABLE_TO_GRANT_JOIN,
         )
 
-    async def check_host_in_room(self, room_id: str, host: str) -> bool:
+    async def is_host_in_room(self, room_id: str, host: str) -> bool:
         with Measure(self._clock, "check_host_in_room"):
```

Comment:
Should the measure name be updated too?

Reply:
I'm tempted to remove the measure block. All the time goes into a database call which should already be measured.. Does that sound reasonable?

Reply:
Err, remind me what measures are used for. Is it just an opentracing span?

Reply:
It's for the 'Per-block Metrics' section in Grafana